imap_processing/cdf/config/imap_lo_l1a_variable_attrs.yaml

```diff
@@ -127,6 +127,7 @@ direct_events:
   CATDESC: Index of number of direct events for pointing
   FIELDNAM: direct_events
   LABLAXIS: DE
+  LABL_PTR_1: direct_events_label
```

Comment:
Do you want to move this upto line 50? It seems like you are inheriting `de_default` a lot and it has `depend_1`. If it has `depend_1`, it also needs `labl_prt_1`. I don't remember if ISTP check catches that or not but it will be something you will come across later.